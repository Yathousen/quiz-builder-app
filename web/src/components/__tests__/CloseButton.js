import React from 'react';
import renderer from 'react-test-renderer';
import { CloseButton } from '../';

it('Close Button', () => {
  expect(renderer.create(<CloseButton />).toJSON()).toMatchSnapshot();
});

it('Disabled Close Button', () => {
  expect(renderer.create(<CloseButton disabled />).toJSON()).toMatchSnapshot();
});
