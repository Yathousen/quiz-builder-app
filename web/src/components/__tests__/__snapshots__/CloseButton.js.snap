// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Close Button 1`] = `
<button
  className="bg-white rounded-md ml-1 p-1 inline-flex items-center justify-center text-gray-400 hover:bg-red-400 hover:text-white focus:outline-none focus:ring-2 focus:ring-inset focus:ring-indigo-500 undefined"
  type="button"
>
  <svg
    aria-hidden="true"
    className="h-4 w-4"
    fill="none"
    stroke="currentColor"
    viewBox="0 0 24 24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M6 18L18 6M6 6l12 12"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
    />
  </svg>
</button>
`;

exports[`Disabled Close Button 1`] = `
<button
  className="bg-white rounded-md ml-1 p-1 inline-flex items-center justify-center text-gray-400 hover:bg-red-400 hover:text-white focus:outline-none focus:ring-2 focus:ring-inset focus:ring-indigo-500 undefined"
  disabled={true}
  type="button"
>
  <svg
    aria-hidden="true"
    className="h-4 w-4"
    fill="none"
    stroke="currentColor"
    viewBox="0 0 24 24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M6 18L18 6M6 6l12 12"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
    />
  </svg>
</button>
`;
